addons/sale/controllers/portal.py

```diff
@@ -154,10 +154,7 @@ def portal_order_page(
                     subtype_xmlid="sale.mt_order_viewed",
                 )
 
-        backend_url = f'/web#model={order_sudo._name}'\
-                      f'&id={order_sudo.id}'\
-                      f'&action={order_sudo._get_portal_return_action().id}'\
-                      f'&view_type=form'
+        backend_url = f'/odoo/action-{order_sudo._get_portal_return_action().id}/{order_sudo.id}'
```

Comment:
I don't find any mention of `sale_subscription` in the task, and the same change should be done there too.
There is a handful of `backend_url` here and there in the code, and I'm not sure all of them are considered in the scope mentioned in your linked task.

Reply:
Also, the link doesn't seem to work well if the website editor is enabled:
![image](https://github.com/user-attachments/assets/674149be-0adf-48e8-8b9f-5baf1c31f56d)

Reply:
The error is not related to the URL modification, the error already exists in master.

There is no mention of any module in the tasks, the PR are created module by module, a PR will be created for also for sale_subscription

Reply:
Didn't take the time to test in master but if it's unrelated, fine :ok_hand:.
For the modules, I thought that you made the table in advance to handle all the scopes, but if you do it step by step forget what I said.